pkgs/misc/drivers/xp-pen-deco-01-v2-driver/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, stdenv, fetchzip, libusb, glibc, libGL, xorg, qtx11extras, wrapQtAppsHook, autoPatchelfHook, ... }:
+
+stdenv.mkDerivation {
+  pname = "xp-pen-deco-01-v2-driver";
```

Comment:
This shouldn't include versioning. 


In addition, when `pname` doesn't match the executables name in `$out/bin` you should set `meta.mainProgram` to avoid breaking `nix run`. That doesn't really apply now, but maybe it will if this is renamed.

Reply:
I don't think this really counts as versioning, since that's the official name of the tablet (the "XP-PEN Deco 01 v2" drawing tablet). It's more like a product name (the "iPhone 12") than a version number.

Reply:
Ah, i misunderstood than. You can ignore my comments about that 👍